### PR TITLE
Allow autoplay and inline media playback in ORKWebViewStep

### PR DESCRIPTION
**Pull request by overcyn**
Updates the webview configuration to allow inline media playback and autoplay.

### CODE REVIEW

File: ResearchKit/Common/ORKWebViewStepViewController.m [hunk 1]
```diff
@@ -55,6 +55,11 @@ - (void)stepDidChange {
     
     if (self.step && [self isViewLoaded]) {
         WKWebViewConfiguration *config = [[WKWebViewConfiguration alloc] init];
+        config.allowsInlineMediaPlayback = true;
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+        config.mediaPlaybackRequiresUserAction = false;
```
Comment (srinathtm-apple) [inline ResearchKit/Common/ORKWebViewStepViewController.m]: Can we avoid using a deprecated API?

Comment (overcyn) [reply ResearchKit/Common/ORKWebViewStepViewController.m]: Were you thinking something like the below, which would work on iOS 8+?

```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
        if ([config respondsToSelector:@selector(mediaTypesRequiringUserActionForPlayback)]) {
            config.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeNone;
        } else if ([config respondsToSelector:@selector(requiresUserActionForMediaPlayback)]) {
            config.requiresUserActionForMediaPlayback = false;
        } else {
            config.mediaPlaybackRequiresUserAction = false;
        }
#pragma clang diagnostic pop
```

Or to just use the latest and support iOS 10+?

```
        if ([config respondsToSelector:@selector(mediaTypesRequiringUserActionForPlayback)]) {
            config.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeNone;
        }
```

Comment (srinathtm-apple) [reply ResearchKit/Common/ORKWebViewStepViewController.m]: I would say use the latest since ResearchKit 2.0 sets the base SDK to iOS 11.0.

Comment (overcyn) [reply ResearchKit/Common/ORKWebViewStepViewController.m]: Fixed

**Comment by srinathtm-apple** [comment]
Merging into master.

**Pull request merged**